pkg/runner/runner_test.go

```diff
@@ -144,12 +145,19 @@ func TestRunEvent(t *testing.T) {
 		// single test for different architecture: linux/arm64
 		{"testdata", "basic", "push", "", platforms, "linux/arm64"},
 	}
+	// 5 of the above tests seem to have problems running on github on a tty
+	tablesTty := []TestJobFileInfo{
+		{"testdata", "commands-waiting-for-input-freeze", "push", "", map[string]string{"ubuntu-latest": "ghcr.io/catthehacker/ubuntu:act-latest"}, ""}, // Need larger image for git freeze test
+	}
 	log.SetLevel(log.DebugLevel)
 
 	ctx := context.Background()
 
 	for _, table := range tables {
-		runTestJobFile(ctx, t, table)
+		runTestJobFile(ctx, t, table, false)
```

Comment:
Enabling tty for all tests seem to have issues on github see https://github.com/nektos/act/runs/5698039825?check_suite_focus=true
> Unrecognized input header: 72

Cannot reproduce locally on my windows 11 pc